plugins/go-build/share/go-build/1.11.0

```diff
@@ -0,0 +1,11 @@
+install_darwin_64bit "Go Darwin 64bit 1.11.0" "https://dl.google.com/go/go1.11.darwin-amd64.tar.gz#12ef48702dc38ce3545152a7eb060c4ac47796212cc10c67f36c4116f96beae4"
```

Comment:
> `#12ef48702dc38ce3545152a7eb060c4ac47796212cc10c67f36c4116f96beae4`

This is probably wrong.  I encountered an error.

```
goenv install 1.11.0
...
checksum mismatch: Go Darwin 64bit 1.11.0.tar.gz (file is corrupt)
expected 12ef48702dc38ce3545152a7eb060c4ac47796212cc10c67f36c4116f96beae4, got 9749e6cb9c6d05cf10445a7c9899b58e72325c54fee9783ed1ac679be8e1e073
```

The following should be correct.

```
install_darwin_64bit "Go Darwin 64bit 1.11.0" "https://dl.google.com/go/go1.11.darwin-amd64.tar.gz#9749e6cb9c6d05cf10445a7c9899b58e72325c54fee9783ed1ac679be8e1e073"
```